src/Relude/Numeric.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+
+{- |
+Copyright: (c) 2018 Kowainik
+License:    MIT
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+-}
+
+module Relude.Numeric
+       ( module Data.Bits
+       , module Data.Int
+       , module Data.Word
+       , module Numeric
+       , module GHC.Float
+       , module GHC.Num
+       , module GHC.Real
+       , integerToBounded
+       , integerToNatural
+       )
+       where
+
+import Numeric
```

Comment:
Hmm, I don't remember this module being exported earlier 🤔 
And I never heard of it. I see that it contains some useful stuff. But probably it's better to export things from this module under separate issue where we can discuss what should be exported from there.